src/commcare_cloud/ansible/deploy_hq.yml

```diff
@@ -0,0 +1,15 @@
+# NOTE this playbook can be used to deploy a new version of commcare-hq. The
+# similarly named deploy_commcarehq.yml playbook is used to configure machines
+# to run commcare-hq, and is expected to be run less frequently than this one.
+
+- name: Deploy CommCare HQ
```

Comment:
I think it should mention something about "hq" to differentiate it from the formplayer deploy. However, the name doesn't matter much since it will usually be called by the "deploy commcare" sub-command of commcare-cloud. Why do you think it should be less similar to `deploy_commcarehq`?

Reply:
Yea, maybe it's not important - I can just see people getting confused when running ansible:

```shell
cchq env ap deploy_[hq|commcarehq].yml
```